server/sonar-web/src/main/js/apps/projectActivity/__tests__/utils-test.js

```diff
@@ -21,13 +21,17 @@
 import * as utils from '../utils';
 
 const ANALYSES = [
-  { key: 'AVyMjlK1HjR_PLDzRbB9', date: new Date('2017-06-09T13:06:10+0200'), events: [] },
```

Comment:
I updated it because the first analysis is supposed to always have a version event and it was not the case.